pom.xml

```diff
@@ -98,7 +98,11 @@
 			<artifactId>spring-boot-configuration-processor</artifactId>
 			<optional>true</optional>
 		</dependency>
-
+		<dependency>
+			<groupId>net.sf.supercsv</groupId>
+			<artifactId>super-csv</artifactId>
+			<version>2.4.0</version>
+		</dependency>
```

Comment:
Why is this added?

Reply:
This was added by accident, this is for the IBP27

Reply:
Delete it for now.

Reply:
I deleted it